Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM node:7
-WORKDIR /app
-COPY package.json /app
-RUN npm install
-COPY . /app
-CMD node merge.js -o merged.json WES-swagger-template.json http://wf-search.light.overture.bio/v2/api-docs http://wf-management.light.overture.bio/v2/api-docs && node app.js
+FROM node:10
+WORKDIR /usr/src/app
+COPY package*.json ./
```

Comment:
we probably don't need this cause there's a `COPY . ./` below eh :P

if you want docker to ignore certain things, a `.dockerignore` file would be the way to go ;)

Reply:
+ 1 on this, I think you can just move the copy below to here and use `.dockerignore`